README.rst

```diff
@@ -0,0 +1,484 @@
+############################################
+                  KWIVER
+############################################
+
+Kitware Image and Video Exploitation and Retrieval
+
+
+The KWIVER toolkit is a collection of software tools designed to
+tackle challenging image and video analysis problems and other related
+challenges. Recently started by Kitware’s Computer Vision and
+Scientific Visualization teams, KWIVER is an ongoing effort to
+transition technology developed over multiple years to the open source
+domain to further research, collaboration, and product development.
+
+The project is structured with the parent `kwiver` repository working
+as CMake "super-build" that pulls in a number of KWIVER and other open
+source projects.
+
+
+Building KWIVER
+===============
+
+Fletch
```

Comment:
This whole section should be largely removed and replaced with a URL to the Fletch page on GIthub.  The only thing needed here are a couple of high-level comments about why we use Fletch and info about how your choice of Fletch packages impacts what you can build in KWIVER.

Any specific instructions about how to build Fletch should be moved to the Fletch readme if they are not already covered there.